Makefile

```
@@ -33,8 +33,8 @@ KUBEBUILDER_ASSETS_PATH := $(dir $(abspath $(firstword $(MAKEFILE_LIST))))bin/ku
 KIND_VERSION=v0.18.0
 HELM_VERSION=v3.11.2
 # This kubectl version supports -k for kustomization.
-KUBECTL_VERSION=v1.30.4
-ENVTEST_K8S_VERSION=1.30.0
+KUBECTL_VERSION=v1.31.1
+ENVTEST_K8S_VERSION=1.31.1
```

```suggestion
ENVTEST_K8S_VERSION=1.31.0
```

It seems that the latest version is v1.30.0:

```
setup-envtest list
(installed)  v1.31.0  darwin/arm64
(installed)  v1.30.0  darwin/arm64
(installed)  v1.29.3  darwin/arm64
(installed)  v1.29.1  darwin/arm64
(installed)  v1.28.3  darwin/arm64
(installed)  v1.27.1  darwin/arm64
(available)  v1.30.0  darwin/arm64
(available)  v1.29.3  darwin/arm64
(available)  v1.29.1  darwin/arm64
(available)  v1.29.0  darwin/arm64
(available)  v1.28.3  darwin/arm64
(available)  v1.28.0  darwin/arm64
(available)  v1.27.1  darwin/arm64
(available)  v1.26.1  darwin/arm64
(available)  v1.26.0  darwin/arm64
(available)  v1.25.0  darwin/arm64
(available)  v1.24.2  darwin/arm64
(available)  v1.24.1  darwin/arm64
```

---

done